test/cpp/qps/qps_worker.cc

```diff
@@ -291,11 +292,10 @@ QpsWorker::QpsWorker(int driver_port, int server_port,
   server_ = builder->BuildAndStart();
   if (server_ == nullptr) {
     gpr_log(GPR_ERROR,
-            "QpsWorker: Fail to BuildAndStart(driver_port=%d, server_port=%d)",
+            "QpsWorker: Fail to BuildAndStart(port=%d) (server_port=%d)",
             driver_port, server_port);
   } else {
-    gpr_log(GPR_INFO,
-            "QpsWorker: BuildAndStart(driver_port=%d, server_port=%d) done",
+    gpr_log(GPR_INFO, "QpsWorker: BuildAndStart(port=%d) (server_port=%d)",
```

Comment:
Again, why change this log?

Reply:
This doesn't need to be changed. Reverted them.